### DISCUSSION TITLE
Tag 可以接入现在 github 的 star list 吗?

### DISCUSSION THREAD

**Discussion by fqd511** [riskers/github-plus-extension#6] [feature]
如题

**Comment by riskers** [comment]
> 如题

意思是在 star list 里，直接显示么？

**Reply by fqd511** [reply]
是把star list 作为tag。当然能反向把tag添加进star list 里更好

**Comment by tasty007** [comment]
分组能多加一层吗...
类似浏览器收藏夹那样

**Reply by riskers** [reply]
这个实现起来有点麻烦

从功能上说，我不太想这么做。因为「文件夹式」的分组深了之后，就基本不会再打开了。不如 Tag 来得痛快。
